src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,56 @@
+.nav__list {
+  margin: 0;
+  display: flex;
+  padding: 0;
+  list-style: none;
+}
+
+.nav__link {
+  display: flex;
+  height: 60px;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+  line-height: 60px;
+  margin-left: 20px;
+  white-space: nowrap;
+  transition-duration: 300ms;
+
+  &:link {
+    color: $main-text-color;
+    text-decoration: none;
+  }
+
+  &:visited {
+    color: $main-text-color;
+    text-decoration: none;
+  }
+
+  &:hover {
+    color: $blue-color;
```

Comment:
Ensure that the variable `$blue-color` is defined in your SCSS files. If not, define it to avoid compilation errors.